data/coops.json

```diff
@@ -13,6 +13,24 @@
       "en": "<p>We are two, plus one part-time courier. In January 2019, after a job loss, we decided to launch a bike delivery service in Carpentras, a middle-sized city. This activity gather our two core values : ecology and sport. Today, just five months after starting, we begin to be well-known and appreciated, and are winning more and more new clients.</p><p>Our main activity is delivering groveries to elderly people. Doing this we feel also useful, as it brings autonomy to older people, who otherway would have to go to retirment homes. We are also cycling for florists, cellar masters, brewers and administrations. We are working with people sharing our values. We definitely think that biking generates strong sympathy !</p>"
     }
   },
+  {
+    "name": "Naofood",
+    "url": "https://naofood.fr/",
+    "facebook_url": "https://www.facebook.com/nao.food",
+    "latitude": 47.206472,
+    "longitude": -1.564297,
+    "city": "Nantes",
```

Comment:
The problem here is that it is the `city` which is displayed on the app on the first screen.
It should be `Nantes - Naofood` (for example)